0000-template.md

```diff
@@ -1,6 +1,6 @@
 # Replace this with your awesome RFC title
-
-**Status:** `proposed`
+* Start Date: YYYY-MM-DD
+* RFC PR: https://github.com/open-telemetry/oteps/pull/0000
```

Comment:
this feels like unnecessary friction. Can always get this from git blame.

Reply:
Just the start date, or also the PR link?

Reply:
date is fine, but PR link seems unnecessary

Reply:
I just removed them both for now. Like you say, Github gives us this info easily, no need to replicate it.